src/components/flowchart-primary-toolbar/flowchart-primary-toolbar.js

```diff
@@ -7,7 +7,10 @@ import {
   toggleTextLabels,
   toggleExpandAllPipelines,
```

Comment:
i think you should add this actions/modular-pipelines as well, it's confusing why this is here. and the other expanded ones are in the other